### PR TITLE
Feedback form conversion

### PR DESCRIPTION
**Pull request by glavery**

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -113,12 +117,27 @@
...
+# Name of the Elasticsearch index
+ELASTICSEARCH_INDEX_NAMES = {
+    'search.documents.section': 'section',
+    'search.documents.chapter': 'chapter',
+    'search.documents.heading': 'heading',
+    'search.documents.subheading': 'sub_heading',
+    'search.documents.commodity': 'commodity',
+}
+
+REST_FRAMEWORK = {
```
Comment (trbateman) [inline config/settings/base.py]: Remove as not needed

File: config/settings/base.py [hunk 2]
```diff
@@ -220,6 +241,11 @@
 RULES_OF_ORIGIN_DATA_PATH = APPS_DIR + "/rules_of_origin/data/{0}"
 RULES_OF_ORIGIN_DOCUMENTS_FILE = RULES_OF_ORIGIN_DATA_PATH.format('/reference/group_documents.csv')
 RULES_OF_ORIGIN_GROUPS_FILE = RULES_OF_ORIGIN_DATA_PATH.format('/reference/country_groups_v3.csv')
+SEARCH_DATA_PATH = APPS_DIR + "/search/data/{0}"
+
+TRADE_TARIFF_API = {
+    "BASE_URL": "https://www.trade-tariff.service.gov.uk/api/v2/{0}",
```
Comment (trbateman) [inline config/settings/base.py]: Envar?

File: compose/development/django/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 FROM python:3.6
 
-RUN apt-get update && apt-get install -y wget
+RUN apt-get update && apt-get install -y wget virtualenv
```
Comment (trbateman) [inline compose/development/django/Dockerfile]: use pipenv?

File: config/settings/base.py [hunk 4]
```diff
@@ -31,8 +31,7 @@
 
 SECRET_KEY = os.environ.get('DJANGO_SECRET_KEY', 'a-secret-key')
 
-ADMIN_ENABLED = True
-
+ADMIN_ENABLED = os.environ.get("ADMIN_ENABLED", True)
```
Comment (trbateman) [inline config/settings/base.py]: Disable by default, and override where needed to mitigate against mistakes

File: config/settings/development.py [hunk 5]
```diff
@@ -17,4 +18,31 @@
...
+ES_URL = 'http://es:9200'
+
+ELASTICSEARCH_DSL = {
+    'default': {
+        'hosts': ES_URL
+    },
+}
+
+EMAIL_BACKEND = 'django.core.mail.backends.filebased.EmailBackend'
+EMAIL_FILE_PATH = '/tmp/app-messages'
```
Comment (trbateman) [inline config/settings/development.py]: Best using a path library and making it relative
Also not every machine has /tmp

Comment (glavery) [reply config/settings/development.py]: Will add a task to add the use of pathLib throughout the app as a tidying task and do that in a separate branch. for now will stick with the BASE_DIR var used elsewhere

File: compose/development/django/Dockerfile (hunk 3 above)
Comment (glavery) [reply compose/development/django/Dockerfile]: Will ad pipenv in a separate branch as mentioned earlier

File: config/settings/base.py (hunk 2 above)
Comment (glavery) [reply config/settings/base.py]: Will add a branch later that deals with this issues and other similar application wide

**Pull request merged**